lib/ui/screen-components/protected/home/restaurants/main/index.tsx

```
@@ -0,0 +1,47 @@
+  return (
+    <>
+      <div className="ml-8 mr-10 md:ml-12 md:mr-14 mb-20">
+        <span className="font-inter font-bold text-2xl leading-8 tracking-normal text-gray-900">
+          All Restaurants
+        </span>
+
+        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 mt-4 justify-items-center items-center">
+          {restaurantsData?.map((item) => {
+            return (
+              <>
+                <Card item={item} isDiscovery={false} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)